.travis.yml

```diff
@@ -8,19 +8,26 @@ go:
 # Don't test developer branches
 branches:
   only:
-  - master
+    - master
 
 # Test only on the most recent stable release.
 matrix:
   include:
-  - os: linux
-    env:
-    - BUILD_MODE=e2e
-  - os: linux
-    env:
-    - BUILD_MODE=test
-    # Coveralls.io token
-    - secure: "TDm6L+hiTIs1CP3sm3eBxfcz7MgDN6nAb4+TbPiQOY6xjaVd53J1fvi9orcQZtvacWIh4ZFSxkCggwzezmIp6ZEad/6kVdCfwmbYTK2jEdx9eAxF6dnX3nNOJLiGqws6RPZB81byqIZD269Llrq36Zq5Xwc9mm9kbNFfHbDfAvPTw5VSkbrV6LTH452U5apkIcapbhI/Ll/aa3uVoDH8lbeudA0V5reTBP+BSNob/FrLjXEpkbpW3QzqzZm+RWBM5xKqIX51Vd5IkwYQZGi9mwf92c+9FcolNjhdFajJBWDEU9WUZyv/wWkijUV2aTj2RYKF2QEwJO2EbDsJLJHQvi1l8jD2xu9Pa+ef0Ia/Js4KkLr9TB9m6tY+3ShMUvxQGfKeGeAmvzYDVPfo2Cu0qTO01nClMZfZM3rM9Sm9l7kN3LaZUrvLDl/kvQVMCMvzcypbcWfI6Eh5uw5WMUjH4QgBxUUaaI6fVFK6dTsmdGYtbiDPqkUMKT/45UpbMoxKrHYB7GSP1lLR6CfifOa9p91Tpcwq+CsOl78do6KEzcKZWA0ojMMAWX9PDtLPr4TwXN9ENDH/IityQAI3cCwfV1LmUZNx0HffznoG7v+bnvi4NfNumUMSmOE+HobDO4vPmW2rpHWprA4AQO3lY/fwZatzhH933maQRH4FRBQfw2M="
+    - os: linux
+      env:
+        - BUILD_MODE=e2e
+    - os: linux
+      env:
+        - BUILD_MODE=test
+        # Coveralls.io token
+        - secure: "TDm6L+hiTIs1CP3sm3eBxfcz7MgDN6nAb4+TbPiQOY6xjaVd53J1fvi9orcQZtvacWIh4ZFSxkCggwzezmIp6ZEad/6kVdCfwmbYTK2jEdx9eAxF6dnX3nNOJLiGqws6RPZB81byqIZD269Llrq36Zq5Xwc9mm9kbNFfHbDfAvPTw5VSkbrV6LTH452U5apkIcapbhI/Ll/aa3uVoDH8lbeudA0V5reTBP+BSNob/FrLjXEpkbpW3QzqzZm+RWBM5xKqIX51Vd5IkwYQZGi9mwf92c+9FcolNjhdFajJBWDEU9WUZyv/wWkijUV2aTj2RYKF2QEwJO2EbDsJLJHQvi1l8jD2xu9Pa+ef0Ia/Js4KkLr9TB9m6tY+3ShMUvxQGfKeGeAmvzYDVPfo2Cu0qTO01nClMZfZM3rM9Sm9l7kN3LaZUrvLDl/kvQVMCMvzcypbcWfI6Eh5uw5WMUjH4QgBxUUaaI6fVFK6dTsmdGYtbiDPqkUMKT/45UpbMoxKrHYB7GSP1lLR6CfifOa9p91Tpcwq+CsOl78do6KEzcKZWA0ojMMAWX9PDtLPr4TwXN9ENDH/IityQAI3cCwfV1LmUZNx0HffznoG7v+bnvi4NfNumUMSmOE+HobDO4vPmW2rpHWprA4AQO3lY/fwZatzhH933maQRH4FRBQfw2M="
```

Comment:
It fails because of the lack of some go testing files and there are some files that have been left unfinished. As now we don't have enough hands on checking those files. But when that is fixed, the CI will have a pass on build.